DISPLAY "(null)" invalid; disabling X11 forwarding
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcublas.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcudnn.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcufft.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcuda.so locally
I tensorflow/stream_executor/dso_loader.cc:108] successfully opened CUDA library libcurand.so locally
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
E tensorflow/stream_executor/cuda/cuda_driver.cc:491] failed call to cuInit: CUDA_ERROR_NO_DEVICE
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:153] retrieving CUDA diagnostic information for host: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:160] hostname: raiders3
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:185] libcuda reported version is: Not found: was unable to find libcuda.so DSO loaded into this program
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:347] driver version file contents: """NVRM version: NVIDIA UNIX x86_64 Kernel Module  367.27  Thu Jun  9 18:53:27 PDT 2016
GCC version:  gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.3) 
"""
I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:189] kernel reported version is: 367.27.0
I tensorflow/core/common_runtime/gpu/gpu_init.cc:81] No GPU devices available on machine.
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, localhost:2210, raiders3:2211, raiders3:2212, raiders8:2213, raiders8:2214, raiders8:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2210
INFO:tensorflow:Learning rate: 0.002500, momentum: 0.900000
INFO:tensorflow:2016-07-27 03:27:36.857929 Supervisor
E0727 03:27:49.149210358   15150 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2205': timeout occurred
E0727 03:27:49.149271738   15150 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 03:27:49.149287645   15150 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2207': timeout occurred
E0727 03:27:49.149300690   15150 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2208': timeout occurred
E0727 03:27:49.673272095   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:27:49.673324724   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:27:49.673339101   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:27:49.673350733   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:27:54.429173467   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:27:54.429216462   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:27:54.429235051   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:27:54.429285449   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:27:57.429113614   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:27:57.429148451   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:28:00.429072400   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:28:00.429106217   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:28:03.429083381   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:28:06.709044421   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:28:09.709062922   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:28:13.277014923   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:28:16.277002882   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:28:19.789078113   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:28:26.981077953   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:28:33.553012471   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:28:36.553080847   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 03:28:36.553126316   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:29:00.109028088   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 03:29:00.109068438   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 03:29:03.321093932   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 03:29:06.321042162   15150 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
run-expr-32node.sh: line 23: 14500 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
